addons/payment_mercado_pago/models/payment_transaction.py

```diff
@@ -170,3 +178,18 @@ def _process_notification_data(self, notification_data):
             self._set_error(
                 "Mercado Pago: " + _("Received data with invalid status: %s", payment_status)
             )
+
+    @api.model
+    def _mercado_pago_response_msg(self, notification_data):
+        """ Return the error message corresponding to the payment status.
+
+        :return: The error message.
+        :param dict notification_data: The notification data sent by the provider.
+        """
+
+        status = notification_data.get('status_detail', 'cc_rejected_other_reason')
+        try:
+            message = "Mercado Pago: " + ERROR_MESSAGE_MAPPING[status]
+        except KeyError:
+            message = "Mercado Pago " + _("could not process this payment. Error code: %s", status)
```

Comment:
Instead of having two fallbacks we could just reuse the `cc_rejected_other_reason` message if the received `status_detail` matches no message.

Reply:
I was seeing it more like a: "a new state was added, we might want to know it"

Reply:
We'll never know :D